src/lib/components/ContentfulRichText/ContentfulRichText.svelte

```diff
@@ -1,16 +1,10 @@
 <script lang="ts">
   import Node from "./nodes/Node.svelte";
-  import type { Document } from "@contentful/rich-text-types";
-  import { markdownDocument } from "./__tests__/documents";
-  import { isDocument } from "./predicates";
+  import type { TopLevelBlock } from "@contentful/rich-text-types";
 
-  export let document: Document = markdownDocument.document;
-
-  if (!isDocument(document)) {
-    throw new Error("document prop is not a document");
-  }
```

Comment:
Moved this logic into `src/routes/test-contentful-content/+page.server.ts`, but we may want to move to some sort of utility function in the future.

Reply:
Moved things back in f22c49c.